src/mesh/mesh_generation.C

```diff
@@ -1406,7 +1406,8 @@ void MeshTools::Generation::build_cube(UnstructuredMesh & mesh,
 
 
   // Done building the mesh.  Now prepare it for use.
-  mesh.prepare_for_use (/*skip_renumber =*/ false);
+  mesh.allow_find_neighbors(true);
```

Comment:
Based on `_skip_find_neighbors(false)` being the default, `allow_find_neighbors(true)` is the default right? Do we need this extra line? As far as I can tell this `prepare_for_use()` call was using all default args before, so calling it with no args should be equivalent...

Reply:
This particular line may be unnecessary, it's true. But I got paranoid after I discovered a libMesh test case where I think we did an initial `prepare_for_use(blah, skip_find_neighbors=true)` and then we performed mesh refinement later and called `prepare_for_use()` in which we relied on the implicit setting of `skip_find_neighbors=false`, so I went through libMesh and any place where we called `prepare_for_use` without any arguments (previously relying on the implicit setting of `skip_find_neighbors=false`), I added this line ahead of it to make sure that the previous behavior was replicated

Reply:
To be clear, calling with no args after this PR is generally not equivalent to calling with no args before this PR.

Reply:
OK, paranoid (and more expressive in general) is always better when there's any chance of ambiguity.